mantle/kola/harness.go

```diff
@@ -314,14 +314,21 @@ type DenyListObj struct {
 	Arches     []string `yaml:"arches"`
 	Platforms  []string `yaml:"platforms"`
 	SnoozeDate string   `yaml:"snooze"`
+	Variant    []string `yaml:"variant"`
 }
 
 type ManifestData struct {
 	Variables struct {
-		Stream string `yaml:"stream"`
+		Stream  string `yaml:"stream"`
```

Comment:
Just small question, where is `stream` used for? I can not find any definition in `manifest.yaml`

Reply:
It's used in Fedora CoreOS only for now.

Reply:
Understand, thanks!